cfme/tests/cloud/test_provisioning.py

```diff
@@ -71,6 +71,7 @@ def vm_name(request):
     return vm_name
 
 
+@pytest.mark.requirement('provision')
```

Comment:
Please can this be replaced wit hthe newer marking method mentioned in the latest email?